### DISCUSSION TITLE
Ender 3V3 problem with {BELTS_SHAPER_CALIBRATION}

### DISCUSSION THREAD

**Discussion by Kpa6ac** [Guilouz/Creality-Helper-Script-Wiki#630] [Issues]
hello, there is a problem with graph generation when executing the {BELTS_SHAPER_CALIBRATION} script. upon completion of execution, the PNG file is not displayed.
![image](https://github.com/user-attachments/assets/50d68301-7af1-4f18-88fa-932f3ae027cd)
![image](https://github.com/user-attachments/assets/48f19e24-cadb-4edc-9fd8-f5016ed61d60)

**Comment by robomopa** [comment]
Issue confirmed, for k1m with the last update .png is still not created.
